@navikt/core/react/src/inline-message/root/InlineMessage.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { forwardRef } from "react";
+import { useRenameCSS, useThemeInternal } from "../../theme/Theme";
+import type { AkselColor } from "../../types";
+import { BodyShort } from "../../typography";
+import type { OverridableComponent } from "../../util";
+import { InlineMessageIcon } from "../icon/InlineMessageIcon";
+
+interface InlineMessageProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * InlineMessage variant.
+   */
+  variant: "info" | "success" | "warning" | "error";
+  /**
+   * InlineMessage size.
+   * @default "medium"
+   */
+  size?: "medium" | "small";
+}
+
+/**
+ * InlineMessage is used to display important messages with other content.
+ * @see [📝 Documentation](https://aksel.nav.no/komponenter/core/inline-message)
+ * @see 🏷️ {@link InlineMessageProps}
+ * @see [🤖 OverridableComponent](https://aksel.nav.no/grunnleggende/kode/overridablecomponent) support
+ * @example
+ * ```jsx
+ *  <InlineMessage variant="error">
+ *    Inline Errormessage
+ *  </InlineMessage>
+ * ```
+ *
+ * @example As a link
+ * ```jsx
+ *  <InlineMessage variant="error" as={Link} href="#">
+ *    Inline Errormessage
+ *  </InlineMessage>
+ * ```
+ */
+const InlineMessage: OverridableComponent<
+  InlineMessageProps,
+  HTMLAnchorElement
```

Comment:
Looks like it doesn't matter in practice (?), but it's technically a div by default

```suggestion
  HTMLDivElement
```